lib/protoboeuf/google/protobuf/any.rb

```diff
@@ -553,10 +553,26 @@ def _encode(buff)
 
         def to_h
           result = {}
-          result["type_url".to_sym] = @type_url
-          result["value".to_sym] = @value
+
+          result[:"type_url"] = @type_url
+          result[:"value"] = @value
+
+          result
+        end
+
+        def as_json(options = {})
+          result = {}
+
+          result["typeUrl"] = @type_url
+          result["value"] = @value
+
           result
         end
+
+        def to_json(options = {})
+          require "json"
+          JSON.dump(as_json(options))
```

Comment:
Can we rename `options` to `as_json_options` please? I would otherwise think `options` applies to formatting the JSON output.